### ISSUE TITLE
Server metadata contains format "application/xml+json"

### ISSUE THREAD

**Issue by pcvolkmer** [samply/blaze#1619]
In Blaze version 0.25.0 the metadata contains

```
{
    ...
    "format": [
        "application/fhir+json"
        "application/xml+json"
    ],
    ...
}
```

Shouldn't the latter format be `application/fhir+xml` instead of `application/xml+json` which makes no sense?

**Comment by alexanderkiel** [comment]
Thanks for finding this. Will release the fix together with the other changes in v0.26 properly tomorrow.

**Comment by pcvolkmer** [comment]
Fixed in #1621
